vendor/modules.txt

```diff
@@ -145,6 +145,7 @@ github.com/dimchansky/utfbom
 github.com/docker/spdystream
 github.com/docker/spdystream/spdy
 # github.com/emicklei/go-restful v2.9.5+incompatible
+## explicit
```

Comment:
Was this a comment you added or something go did?

Reply:
Nope. It was added by tooling.